go/examples/vaccine_test.go

```diff
@@ -0,0 +1,142 @@
+package examples
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"github.com/stretchr/testify/assert"
+	wallet "github.com/trinsic-id/sdk/go/proto/universalwallet/v1"
+	template "github.com/trinsic-id/sdk/go/proto/verifiablecredentials/templates/v1"
+	credential "github.com/trinsic-id/sdk/go/proto/verifiablecredentials/v1"
+	"github.com/trinsic-id/sdk/go/services"
+	"testing"
+)
+
+func createTemplate(templateService services.CredentialTemplateService, t *testing.T) *template.TemplateData {
+	// createTemplate() {
+	templateRequest := &template.CreateCredentialTemplateRequest{Name: "VaccinationCertificate", AllowAdditionalFields: false, Fields: make(map[string]*template.TemplateField)}
+	templateRequest.Fields["firstName"] = &template.TemplateField{Description: "Given name"}
```

Comment:
```suggestion
	templateRequest.Fields["firstName"] = &template.TemplateField{Description: "First name of vaccine recipient"}
```

To ensure template parity across languages